src/Game.cxx

```diff
@@ -286,6 +286,28 @@ void Game::run(bool SkipMenu)
 #ifdef USE_AUDIO
   if (!Settings::instance().audio3DStatus)
   {
+	  /*
+	StandardReverbProperties temp_rev_prop = {
+		1.0,	//flDensity
+		1.0,	//flDiffusion
+		0.8,	//flGain
+		0.89,	//flGainHF
+		3.00,	//flDecayTime
+		0.83,	//flDecayHFRatio
+		0.05,	//flReflectionsGain
+		0.007,	//flReflectionsDelay
+		1.26,	//flLateReverbGain
+		0.011,	//flLateReverbDelay
+		0.994,	//flAirAbsorptionGainHF
+		0.0	//flRoomRolloffFactor
+	};
+	*/
```

Comment:
is this documentation? IF yes, please move it to the header / doxygen so it is shown in doxygen documentation. 
if not, please remove it.